packages/fungible-token/bridge-fungible-token/tests/utils/builder.rs

```diff
@@ -41,35 +48,20 @@ pub async fn build_contract_message_tx(
 
     // Build a change output for the owner of the first provided coin input
     if !gas_coins.is_empty() {
```

Comment:
No more outputs being added below?

Reply:
No asset change generated in these file, so yeah, no outputs